composer.json

```diff
@@ -1,12 +1,17 @@
 {
 	"name": "automattic/wp-calypso",
-	"require": {
-		"wp-phpunit/wp-phpunit": "^5.4",
-		"automattic/jetpack-codesniffer": "^2.1"
-	},
 	"config": {
 		"allow-plugins": {
 			"dealerdirect/phpcodesniffer-composer-installer": true
+		},
+		"platform": {
+			"php": "8.2"
 		}
+	},
+	"require-dev": {
+		"automattic/jetpack-codesniffer": "^2.1",
+		"phpunit/phpunit": "^9",
```

Comment:
The wp-env update basically meant that you should manage your own phpunit version instead of bundling it, so we do that here.